taca/analysis/analysis.py

```diff
@@ -40,11 +40,11 @@ def _run_type(run):
         #this information about the run type (with HiSeq2.5 applicationaName does not work anymore, but as for a long time we will have instruments not updated I need to find out somehting that works
         try:
             #Works for recent control software
-            runtype=rp.data['RunParameters']["Setup"].get("Flowcell")
+            runtype=rp.data['RunParameters']["Setup"]["Flowcell"]
         except KeyError:
             #use this as second resource but print a warning in the logs
             logger.warn("Parsing runParameters to fecth instrument type, not found Flowcell information in it. Using ApplicaiotnName")
-            runtype=rp.data.get("ApplicationName")
```

Comment:
This is a serious change, the default was
rp.data.get("ApplicationName")
and now it is 
rp.data['RunParameters']["Setup"].get("ApplicationName", "")
It's not the same value anymore, and now it can get empty string as a value. Why empty string over None ? Are you sure that the RunParameters always have ApplicationName under RunParameters and Setup ?